university-task-02/src/pages/Common/SocialLinks/SocialLinks.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const SocialLink = ({ url, iconName, path }) => {
```

Comment:
As we usually have not more then 3 - 5 social network attached to the project it's better to have the separate component for each of them and use them of necessity